source/common/buffer/buffer_impl.h

```diff
@@ -307,6 +314,7 @@ class Slice {
   void transferDrainTrackersTo(Slice& destination) {
     destination.drain_trackers_.splice(destination.drain_trackers_.end(), drain_trackers_);
     ASSERT(drain_trackers_.empty());
+    destination.releasor_.swap(releasor_);
```

Comment:
what happens with the original releasor that the destination has? Is it fine to toss it as we're doing? ISTM that we don't need to do this especially as the only place `transferDrainTrackersTo` is called is when we copy the current slice into destination in `coalesceOrAddSlice`.

Reply:
yes, you are right, I shouldn't change releasor here.